qualtran/serialization/resolver_dict.py

```diff
@@ -248,6 +247,7 @@
     "qualtran.bloqs.data_loading.qrom.QROM": qualtran.bloqs.data_loading.qrom.QROM,
     "qualtran.bloqs.data_loading.select_swap_qrom.SelectSwapQROM": qualtran.bloqs.data_loading.select_swap_qrom.SelectSwapQROM,
     "qualtran.bloqs.mod_arithmetic.CModAddK": qualtran.bloqs.mod_arithmetic.CModAddK,
+    "qualtran.bloqs.mod_arithmetic.mod_addition.ModAddK": qualtran.bloqs.mod_arithmetic.mod_addition.ModAddK,
```

Comment:
Is this intended as part of the PR ?

Reply:
PrepareHubbard wouldn't serialized without it, I think the module name was slightly off before.

Reply:
In particular, ModAddK moved to mod_arithmetic from arithmetic.